hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -1587,6 +1663,31 @@ private List<String> createFiles(String parent,
     return keyNames;
   }
 
+  private void createFile(String bucketName, String keyName)
+      throws IOException {
+    OmKeyArgs keyArgs = createBuilder(bucketName).setKeyName(keyName).build();
+    OpenKeySession keySession = writeClient.openKey(keyArgs);
+    keyArgs.setLocationInfoList(
+        keySession.getKeyInfo().getLatestVersionLocations().getLocationList());
+    writeClient.commitKey(keyArgs, keySession.getId());
+  }
+
+  private void assertFileNotFound(String bucketName, String keyName)
+      throws IOException {
+    OmKeyArgs keyArgs = createBuilder(bucketName).setKeyName(keyName).build();
+    OMException ex = Assert.assertThrows(OMException.class,
+        () -> keyManager.getFileStatus(keyArgs));
+    Assert.assertEquals(OMException.ResultCodes.FILE_NOT_FOUND, ex.getResult());
+  }
+
+  private void assertIsDirectory(String bucketName, String keyName)
+      throws IOException {
+    OmKeyArgs keyArgs = createBuilder(bucketName).setKeyName(keyName).build();
+    OzoneFileStatus ozoneFileStatus = keyManager.getFileStatus(keyArgs);
+    Assert.assertEquals(keyName, ozoneFileStatus.getKeyInfo().getFileName());
```

Comment:
We should assert `volumeName` and `bucketName`.

Run the test few times and make sure that it passes all the time.